### ISSUE TITLE
some issue with setup and teardown in pytest suite

### ISSUE THREAD

**Issue by pytestbot** [pytest-dev/pytest#712]
Originally reported by: **BitBucket: [tibor_arpas](http://bitbucket.org/tibor_arpas), GitHub: [tibor_arpas](http://github.com/tibor_arpas)**

----------------------------------------

I'm on platform 'darwin'. I select only one test, which by chance is skipped. I get an error instead of a success.

```
#!console

py.test testing/test_argcomplete.py::TestArgComplete::test_compare_with_compgen
============================= test session starts ==============================
platform darwin -- Python 2.7.8 -- py-1.4.26 -- pytest-2.8.0.dev1
rootdir: /Users/tibor/tmonworkspace/pytest, inifile: tox.ini
collected 3 items 

testing/test_argcomplete.py sE

==================================== ERRORS ====================================
________ ERROR at teardown of TestArgComplete.test_compare_with_compgen ________

item = <Function 'test_compare_with_compgen'>
__multicall__ = <MultiCall 0 results, 2 meths, kwargs={'item': <Function 'test_compare_with_compgen'>, '__multicall__': <MultiCall 0 results, 2 meths, kwargs={...}>, 'nextitem': None}>

    def pytest_runtest_teardown(item, __multicall__):
>       item.config._basedir.chdir()
E       AttributeError: 'Config' object has no attribute '_basedir'

testing/conftest.py:70: AttributeError
=========================== short test summary info ============================
SKIP [1] /Users/tibor/tmonworkspace/pytest/_pytest/skipping.py:140: condition: sys.platform in ('win32', 'darwin')
```

----------------------------------------
- Bitbucket: https://bitbucket.org/pytest-dev/pytest/issue/712

**Comment by tarpas** [comment]
Actually I reported this. I could fix this by patching pytest_runtest_teardown, but in master I don't see the conftest.py at all, so I guess this can be closed.

**Comment by blueyed** [comment]
This appears to have been changed/addressed in a8afba0.

Closing as per initial reporter.
